packages/react-components/react-components/src/index.ts

```diff
@@ -189,7 +319,9 @@ export type {
 } from '@fluentui/react-accordion';
 export {
   Avatar,
+  // exported from component, but not exported from stable or unstable //AvatarGroup, et.al
```

Comment:
AFAIR this is as expected.

Pattern should be like following (please correct me if I'm wrong) :
- export "everything" from particular package (eg react-text)
- export only what is really public API from `react-components` both stable and `unstable`

would be great to have this documented